src/tasks/tasks.service.ts

```
@@ -285,109 +287,151 @@ export class TasksService {
+            )
+          : await this.ensureCategoryMembership(
+              contactId,
+              dto.groupId,
+              GroupCategoryPurpose.SERVING_TEAM,
+              true,
+            );
+      membershipId = savedMembership.id;
+    }
+
+    // contact_activity insert and task status update are atomic
+    await this.connection.transaction(async (em: EntityManager) => {
```

ESLint

[error] Unexpected space before function parentheses.

(space-before-function-paren)